flake/modules/home-manager/default.nix

```
@@ -68,6 +68,16 @@ in {
   meta.maintainers = with lib.maintainers; [sioodmy NotAShelf];
   imports = [./options];
   config = mkIf cfg.enable {
+    assertions = [
+      {
+        assertion = hasInfix "esr" cfg.package.version;
+        message = ''
+          Package ${cfg.package.pname} is not an ESR release of firefox.
+          The package provided to schizofox must be an ESR release, due to policy management differences.
+        '';
+      }
+    ];
```

```suggestion
    assertions = [
      {
        assertion = hasInfix "esr" cfg.package.version;
        message = ''
          The package provided to 'programs.schizofox.package' is not an ESR release of Firefox: ${cfg.package.pname}
          
          For policies to function as intended, you must an ESR release of Firefox. If you think this is a mistake, please open an issue.
        '';
      }
    ];
```

I feel like this could've been a warning, but an assertion is the more aggressive and I feel like the more precise option. Do we rather some things silently get ignored, or eval fails to warn you. Assertion is good enough, but I've requested changes to make the wording more clear.